src/network/receiver.rs

```diff
@@ -1,40 +1,51 @@
 // Copyright(C) Facebook, Inc. and its affiliates.
-use super::error::NetworkError;
```

Comment:
these receiver code seems to be missing the last changes I made in this commit:

https://github.com/Entropy1729/consensus-workbench/pull/24/commits/55afaa223e125ff95d33a0c5dc04ae67ff933f9a